src/main/java/org/sagebionetworks/bridge/exporter3/results/AssessmentResultSummarizer.kt

```diff
@@ -0,0 +1,70 @@
+package org.sagebionetworks.bridge.exporter3.results
+
+import org.sagebionetworks.assessmentmodel.AnswerColumn
+import org.sagebionetworks.assessmentmodel.AssessmentResult
+import org.sagebionetworks.assessmentmodel.serialization.Serialization
+import org.sagebionetworks.assessmentmodel.toFlatAnswers
+import org.sagebionetworks.assessmentmodel.toFlatAnswersDefinition
+import org.sagebionetworks.bridge.rest.RestUtils
+import org.sagebionetworks.bridge.rest.model.Assessment
+import org.sagebionetworks.bridge.rest.model.AssessmentConfig
+import org.slf4j.LoggerFactory
+
+class AssessmentResultSummarizer(private val assessment: Assessment, private val assessmentConfig: AssessmentConfig): AssessmentSummarizer {
+
+    init {
+        assert(assessment.frameworkIdentifier == FRAMEWORK_IDENTIFIER)
+    }
+
+    override val resultFilename: String
+        get() = "assessmentResult.json"
+
+    override fun canSummarize(assessment: Assessment): Boolean {
+        return assessment.frameworkIdentifier == "health.bridgedigital.assessment"
+    }
+
+    /**
+     * Returns a flattened map of results for a survey where the key is the column name and the value is a string
+     * representation of the result.
+     */
+    override fun summarizeResults(resultJson: String): Map<String, String> {
+        val assessmentResult: AssessmentResult = Serialization.JsonCoder.default.decodeFromString(resultJson)
+        val answers = assessmentResult.toFlatAnswers()
+        val columnNames = getColumnNames()
+        for (column in answers.keys) {
+            if (!columnNames.contains(column)) {
+                // TODO: How should we best log any unexpected columns? -nbrown 11/2/23
+                LOG.debug("Unexpected column: " + column + " when summarizing results for assessment: " + assessment.identifier)
```

Comment:
We should log this either as an info or a warning. Debug doesn't show up in our logs. Info we can make dashboards for, but there are no alarms. Warning will alarm if there are more than 100 per hour.

We should also include the app ID and record ID so we can trace the data through the logs.